lib/Model/Importer/Binary/PECOFFImporter.cpp

```diff
@@ -154,13 +160,92 @@ void PECOFFImporter::parseSymbols() {
   }
 }
 
+void PECOFFImporter::recordImportedFunction(
+  iterator_range<llvm::object::imported_symbol_iterator> Range,
+  uint32_t ImportAddressTableAddr) {
+  using namespace llvm::object;
+
+  // Index of entries within import table.
+  uint32_t Index = 0;
```

Comment:
Consider using `llvm::enumerate`.

Reply:
Hmm... I don't think the `llvm::enumerate` is appropriate for this.